packages/web-react/src/components/Pagination/UncontrolledPagination.tsx

```diff
@@ -15,13 +15,15 @@ const UncontrolledPagination = (props: SpiritUncontrolledPaginationProps): JSX.E
     accessibilityLabelPrevious = 'Previous',
     accessibilityLabelNext = 'Next',
     defaultPage = 1,
+    href,
```

Comment:
I am absolutely not sure about this approach and usage in PaginationLinkPrevious and PaginationLinkNext.

Reply:
In this case, there is a `useButtonLinkAriaProps` that takes care of adding `role="button"` to the `a` tag when the href is missing. So passing a href here is not necessary, IMHO. You can use anchor without a href but it needs to have the role button.